TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -71,6 +71,10 @@ public MyTaskResults Run()
             WriteQuantificationResults();
             WritePrunedDatabase();
             WritePeptideResults(); // modifies the FDR results for PSMs, so do this last
+            if(Parameters.ProteinList.Any((p => p.GetVariantProteins(4,1).Count()>0)))
```

Comment:
The `(4,1)` should be values from the settings. Max combinitoric variants and minAlleleDepth.

Reply:
using the GetVariantProteins(4,1) was problematic. I have changed it to checking if any proteins have an applied variant and if so write the specified results.

Reply:
That makes sense. This `GetVariantProteins(4,1)` should still return a single `Protein` if there are no variants.